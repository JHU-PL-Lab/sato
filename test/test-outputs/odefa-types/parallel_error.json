[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "b" ],
          "err_match_val": "1",
          "err_match_expected": "bool",
          "err_match_actual": "int"
        }
      ]
    ],
    "err_input_seq": [ 1 ],
    "err_location": "rg = a or b",
    "err_steps": 427
  },
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "a" ],
          "err_match_val": "true",
          "err_match_expected": "int",
          "err_match_actual": "bool"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "rf = a + b",
    "err_steps": 435
  }
]