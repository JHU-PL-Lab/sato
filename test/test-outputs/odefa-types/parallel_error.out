Type errors for:
- Input sequence  : [0]
- Found at clause : rf = a + b
--------------------
* Value    : a = true
* Expected : int
* Actual   : bool
Type errors for:
- Input sequence  : [1]
- Found at clause : rg = a or b
--------------------
* Value    : b = 1
* Expected : bool
* Actual   : int
2 errors found. No further control flows exist.
